### PR TITLE
Add timestamps to autoinst-log.txt

### PR DESCRIPTION
**Pull request by dzyuzin**

### CODE REVIEW

**Comment by okurz** [comment]
looks nice. Can you provide an example log output how this will look? Also, please see https://github.com/os-autoinst/os-autoinst#rules-for-commits and adjust your commit message accordingly (s/Added/Add/)

**Comment by dzyuzin** [comment]
Sample autoinst-log.txt: http://ix64sap002.qa.suse.de/tests/1001/file/autoinst-log.txt

File: bmwqemu.pm [hunk 1]
```diff
@@ -211,18 +211,23 @@ sub set_ocr_rect {
 
 # util and helper functions
 
+sub get_timestamp {
+    my $t = gettimeofday;
+    return sprintf "%s.%04d ", (POSIX::strftime "%H:%M:%S", gmtime($t)), 10000 * ($t - int($t));
```
Comment (aaannz) [inline bmwqemu.pm]: Is UTC what we want to have in logs, or rather localized timezone? (`gmtime` vs `localtime`).

Comment (dzyuzin) [inline bmwqemu.pm]: modstart() uses gmtime, so I also used gmtime :)

**Comment by coolo** [comment]
why are these Debug: lines not prefixed? I guess it's time we create some useful logging class ;(

**Comment by dzyuzin** [comment]
Debug lines were not prefixed because they don't use print_possibly_colored()

**Comment by okurz** [comment]
output looks like
```
15:13:30.7599 <<< send_key(key='alt-n')
Debug: /var/lib/openqa/share/tests/sle/tests/installation/sles4sap_wizard_nw.pm:19 called testapi::send_key
15:13:30.9636 <<< wait_idle(timeout=19)
15:13:32.9695 wait_idle 0 d=99
15:13:33.9713 wait_idle 0 d=102
15:13:34.9726 wait_idle 0 d=102
15:13:35.9774 wait_idle 0 d=102
```

except for the "Debug" line being lonely, which should be a different issue/PR, this totally LGTM :-)

**Comment by dzyuzin** [comment]
Check http://ix64sap002.qa.suse.de/tests/1005/file/autoinst-log.txt , debug lines are not lonely anymore :)

**Comment by okurz** [comment]
LGTM nice, unsure about any potential "performance impact" but should work

**Pull request merged**